src/components/MapGuide/styled.ts

```diff
@@ -36,42 +33,27 @@ export const StyledMapGuideButton = styled('div')`
     }
 `;
 
-export const StyledMapGuideDialog = styled(Dialog)`
-    .MuiDialog-paperScrollPaper {
-        background: #0094e2;
-        color: #fff;
-    }
-    h1,
+export const StyledTooltipContainer = styled('div')`
     p,
     a {
         color: #fff;
     }
     p {
+        padding: 1.4rem 0;
```

Comment:
If you use `Typography` component like mentioned above then you can apply all those styles directly in the `sx` property